applications/vessel-history/src/features/settings/components/Encounters.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useCallback } from 'react'
+import cx from 'classnames'
+import { MultiSelect, InputText, IconButton } from '@globalfishingwatch/ui-components'
+import '@globalfishingwatch/ui-components/dist/base.css'
+import { SelectOption } from '@globalfishingwatch/ui-components/dist/select'
+import { MultiSelectOption } from '@globalfishingwatch//ui-components/dist/multi-select'
+import { t } from 'features/i18n/i18n'
+import { SettingsEvents } from '../settings.slice'
+import { useApplySettingsConnect } from '../settings.hooks'
+import styles from './SettingsComponents.module.css'
+
+interface SettingsProps {
+  settings: SettingsEvents
+  section: string
+}
+
+const Encounters: React.FC<SettingsProps> = (props): React.ReactElement => {
+  const { settings, section } = props
+
+  const { setSettingOptions, setSetting } = useApplySettingsConnect()
+
+  const EEZ_OPTIONS: SelectOption[] = [
+    {
+      id: 'ghana',
+      label: 'Ghana',
+    },
+    {
+      id: 'argentina',
+      label: 'Argentina',
+    },
+  ]
+
+  const RFMOS_OPTIONS: MultiSelectOption[] = [
+    {
+      id: 'CCSBT',
+      label: 'CCSBT',
+    },
+    {
+      id: 'IATTC',
+      label: 'IATTC',
+    },
+    {
+      id: 'ICCAT',
+      label: 'ICCAT',
+    },
+    {
+      id: 'IOTC',
+      label: 'IOTC',
+    },
+    {
+      id: 'NPFC',
+      label: 'NPFC',
+    },
+    {
+      id: 'SPRFMO',
+      label: 'SPRFMO',
+    },
+    {
+      id: 'WCPFC',
+      label: 'WCPFC',
+    },
+  ]
+
+  const eezs = EEZ_OPTIONS.filter((option) => settings.eezs?.includes(option.id))
+  const rfmos = RFMOS_OPTIONS.filter((option) => settings.rfmos?.includes(option.id))
+  const mpas = EEZ_OPTIONS.filter((option) => settings.mpas?.includes(option.id))
+
+  return (
+    <div className={styles.settingsFieldsContainer}>
+      <div className={styles.settingsField}>
+        <label>
+          Events in There EEZs
```

Comment:
Typo, it should say 'Events in these EEZs'

Reply:
will be fixed in the translation